src/Containers/CadView.jsx

```diff
@@ -452,7 +453,7 @@ export default function CadView({
         const pathIds = getPathIdsForElements(lastId)
         const repoFilePath = modelPath.gitpath ? modelPath.getRepoPath() : modelPath.filepath
         const path = pathIds.join('/')
-        navigate(`${pathPrefix}${repoFilePath}/${path}`)
+        navWith(navigate, `${pathPrefix}${repoFilePath}/${path}`)
```

Comment:
having a camera hash parameter in the url while making a new selection resets camera position to the url parameter, so we might want to clear it somehow to fix this?

Reply:
Done.

Reply:
@ronplaymeta I got a merge conflict with how you're using this for placemarks.  I'm going with my version for now, but just keep in mind we should think about how persistent state is when changing selection if a placemark is active